cmd/handler-utils.go

```diff
@@ -114,16 +114,24 @@ func extractMetadataFromHeader(header http.Header) (map[string]string, error) {
 			metadata[supportedHeader] = header.Get(supportedHeader)
 		}
 	}
+	const MaxHeaderSize = 8 * 1024
```

Comment:
A relevant AWS S3 docs reference URL should also be commented to indicate this spec restriction.